stdlib/test/memory/test_unsafepointer.mojo

```diff
@@ -10,7 +10,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ===----------------------------------------------------------------------=== #
-# RUN: %mojo %s | FileCheck %s --dump-input=always
+# RUN: %mojo -debug-level full %s
```

Comment:
**Suggestion** Drop the `-debug-level full` as in other PRs.

Reply:
Done